adminpages/updates.php

```diff
@@ -20,11 +20,11 @@
 	?>
 	<p id="pmpro_updates_intro"><?php _e('Updates are processing. This may take a few minutes to complete.', 'paid-memberships-pro' );?></p>
 	<p id="pmpro_updates_progress">[...]</p>
-	<textarea id="pmpro_updates_status" rows="10" cols="60">Loading...</textarea>
+	<textarea id="pmpro_updates_status" rows="10" cols="60"><?php _e( 'Loading&hellip;' ) ?></textarea>

```

Comment:
...pods? I thought to add no context in Loading string because it already exists in wp core with the hellip.

Reply:
@mircobabini  The `esc_textarea()` function will render the `&hellip;` as escaped -- essentially without showing the `...`. Should we move the `...` outside of the i18n function or use a placeholder instead? Should we use just a basic `...` here?

Reply:
@mircobabini sorry, typo on the pods part :)

Reply:
Perhaps the only solution here is to go with a different `...` (instead of `&hellip;` because of the context we need to output this as and the best-practice recommendation that we escape HTML in translated strings.

Reply:
Since we are in a textarea yes, I would 100% say the same. Let's go for it.